src/rng.cpp

```diff
@@ -76,5 +76,7 @@ int djb2_hash( const unsigned char *str )
 double rng_normal( double lo, double hi )
 {
     static std::default_random_engine eng;
-    return std::normal_distribution<double>( ( hi + lo ) / 2, 3 )( eng );
+    double range = ( hi + lo );
+    double val = std::normal_distribution<double>( range / 2, range / 4 )( eng );
```

Comment:
You can keep the implementation hidden and still expose the important arguments: mean and standard deviation.
Those are the two qualities of a normal distribution that describe it unambiguously. High and low do not unambiguously describe any normal distribution.